capstone/src/main/java/com/google/sps/data/SurveyResponse.java

```diff
@@ -0,0 +1,68 @@
+package com.google.sps.data;
+
+/* This object represents each response a user submits */
+public class SurveyResponse {
+    private String id;
+    private double score;
+    private String gender;
+    private String ageRange;
+    private long responseTime;
+    private String date; 
```

Comment:
I'd recommend storing date as a `Date` object, java has this available.

Reply:
If it's alright may I add this in my next PR? My next PR is meant for making the synthetic results more similar to actual data